### PR TITLE
Completed Project

### PR DESCRIPTION
**Pull request by Kwaleyela-Ikafa**
### In this branch I:

   - Installed React Redux, Redux Logger, and React Router
   - Wrote actions and reducers for fetching coins.
   - Added coin details page to show more details and display chart statistics using charts.js
   - Added tests functionality for coin component
   - Added tests functionality for Home component
   - [Video presentation of project](https://drive.google.com/file/d/14mKG9ptCVgH3wEt37gsUfth5VnucGCus/view?usp=sharing)

### CODE REVIEW

**Comment by Kwaleyela-Ikafa** [comment]
please ignore Netlify errors, the site is up and running

File: src/Components/Homepage.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <section>
+      <Navbar />
+      <div className="m-5 grid md:grid-cols-4 sm:grid-cols-2  grid-cols-2 gap-2 ">
+        {coins.map((coin) => (
+          <Coin key={coin.uuid} coin={coin} />
+        ))}
+      </div>
+    </section>
```
Comment (Nemwel-Boniface) [inline src/Components/Homepage.js]: - [x] Great job coming up with a design that really matches the provided Behance design. However note that you are required to stick as much as possible to the provided design this included typography and also colors. Your project currently has a white theme this is perfectly fine. However, to meet all the project requirements, you are supposed to choose between the pink theme or the blue theme. Kindly ensure that you make this change.

**This is how your project theme currently looks like:**

image

**This is one of the desired project themes:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Kwaleyela-Ikafa     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your React capstone so far! :clap: It looks awesome :fire: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [x] As requested in the project requirements, when you are taking a demo video of your project, you are required to show a description of the project and also show a piece of code that you are proud of. In your demo video, you did not show these. Kindly retake the presentation video and include it again. The next reviewer will assert that you made this change.

image


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      This HTML file is a template.
+      If you open it directly in the browser, you will see an empty page.
+
+      You can add webfonts, meta tags, or analytics to this file.
+      The build step will place the bundled scripts into the <body> tag.
+
+      To begin the development, run `npm start` or `yarn start`.
+      To create a production bundle, use `npm run build` or `yarn build`.
+    -->
+  </body>
```
Comment (fobadara) [inline public/index.html:42]: [REQUIRED] Good job fixing the colours as suggested by the previous reviewer but you are not quite done yet. Kindly ensure that the shade of blue you use is close to that of the template.

|Template|
|--------|
|image|

[REQUIRED] Kindly take note of the layout and space. They should match the template as much as possible. Your setup is quite pretty but the customer is always right.

|Your Design|
|--------|
|image|

Fix this and you are good to go

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Kwaleyela-Ikafa, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: public/index.html (hunk 2 above)
Comment (Kwaleyela-Ikafa) [reply public/index.html:42]: @fobadara matching the shade of blue used is difficult as they're so many shades of blue

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @Kwaleyela-Ikafa, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**